metadata/com.project_rk.launcher.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <releases>
+    <release version="1.0" date="2024-11-08" />
+  </releases>
+  <id>com.project_rk.launcher</id>
+  
+  <name>Project Rubi-Ka</name>
+  <summary>Launcher for Project Rubi-Ka</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>CC-BY-NC-ND-4.0</project_license>
```

Comment:
where is this license stated? how can I verify?

Reply:
It's in a private hosted git server, it's free to distribute but not open.

Reply:
so not possible to verify?

Reply:
this is not resolved, it seems like a black box to me. You first mis licensed it in the PR description and in metainfo then added a CC license which is not used for code

Now there is no way to verify any information.

Reply:
It's just a privately hosted software project, open to distribute, the project itself is small and there isn't a publicly available source, but we distribute the launcher itself. I can provide download links to our linux launcher, if this somehow satisfies suspicions; I understand CC license doesn't typically fit software; And copyleft isn't really what applies to our use case, are there license restrictions?

Reply:
Have the license stated on the website somewhere https://project-rk.com/pages/about

Reply:
you can just have your own proprietary license with "Licenseref-proprietary" see https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#license, that's what usually most closed source apps do.

Reply:
WIP

Reply:
While not usually used for software, CC-BY-NC-ND-4.0 I think fits the use case best; If we put that on the site for verification, will it still suffice? or is there some flathub policy that restricts the use of specific licenses like CC?

Just trying to decide if its fully necessary to write our own license, or if its sufficient to reference something like the CC to meet attribution requirements here and not have to reinvent the wheel

Reply:
having it on the website is fine.

Reply:
Technically the flatpak would be in violation of the ND clause. Because it creates a derivative.

Beside CC0, CC license aren't in general suited for software.